src/service/poc_lora.proto

```diff
@@ -180,8 +186,16 @@ message gateway_reward_share {
   uint64 start_period = 4;
   /// Unix timestamp in seconds of the end of the reward period
   uint64 end_period = 5;
+  /// type of reward
+  reward_type reward_type = 6;
```

Comment:
You can't just insert a field and renumber. That'll break all the existing files. You will need to add a number at the end.

Reply:
hmm  nothing is renumbered that was already in use

Reply:
<img width="289" alt="Screenshot 2023-04-05 at 7 44 31 AM" src="https://user-images.githubusercontent.com/11398/230084069-1c40c6ff-3bf0-4193-a73c-7405d28763b5.png">

no? Am I not seeing this right?

Reply:
dc_transfer_amount is not yet in use, nothing has been deployed with that